app/src/main/java/sw19/moring03/paint/MainActivity.java

```diff
@@ -241,6 +257,31 @@ public void setChosenColor(int chosenColor) {
         this.chosenColor = chosenColor;
     }
 
+    public void chooseLineType(View view) {
+        MenuItem menuItem = menu.findItem(R.id.toolChooserButton);
+        setChosenTool(Tool.DRAW_LINE);
```

Comment:
This is more a design decision, but: if the line type we choose also applies to shapes, etc. and not just lines, why do we change the tool back to LineTool everytime we change the line type? for me it kinda feels counter-intuitive: I've just chosen to draw a circle and then want to change the line type to "dotted". then I have to re-select the CircleTool again.